addons/payment/models/account_payment_method.py

```diff
@@ -12,32 +12,44 @@ class AccountPaymentMethodLine(models.Model):
     payment_acquirer_id = fields.Many2one(
         comodel_name='payment.acquirer',
         compute='_compute_payment_acquirer_id',
-        store=True
+        store=True,
+        readonly=False,
+        domain="[('provider', '=', code)]",
```

Comment:
FYI it is breaking stable policies since that it requires to update the module using `-u` in order to have the new field code in the view

Is it expected?

@smetl @epictete